### PR TITLE
Add Nuno's dotfiles

### PR DESCRIPTION
**Pull request by nunorafaelrocha**
This PR adds Nuno's dotfiles to this awesome list - A must-have OFC!

### CODE REVIEW

**Comment by LankyLou** [comment]
image

**Review by vvnkr — CHANGES_REQUESTED** [review]
Please don't add Nuno's dotfiles.

File: README.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ Tools, packages, and misc things that DataCamp thinks are awesome. Feel free to
 
 - [GIPHY Capture](https://apps.apple.com/us/app/giphy-capture-the-gif-maker/id668208984?mt=12) - The best way to create GIFs on your Mac.
 - [Bear](https://bear.app/) - Bear is a beautiful, flexible writing app for crafting notes and prose.
+- [dotfiles](https://github.com/nunorafaelrocha/dotfiles) - Your dotfiles are how you personalize your system. These are Nuno's, fork it and boost your productivity :rocket:
```
Comment (vdhpieter) [inline README.md:44]: I would add a better example, these dotfiles you linked are not very good

**Review by vdhpieter — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nunorafaelrocha) [reply README.md:44]: @vdhpieter have you tried them? Can you provide another example?

**Pull request merged**